srcpkgs/python3-RPi.GPIO/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'python3-RPi.GPIO'
+pkgname=python3-RPi.GPIO
+version=0.7.0
+revision=1
+archs="armv[67]l* aarch64*"
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="python3"
+short_desc="Module to control Raspberry Pi GPIO channels"
+maintainer="amak <amak.git@outlook.com>"
+license="MIT"
+homepage="https://sourceforge.net/projects/raspberry-gpio-python"
+distfiles="${SOURCEFORGE_SITE}/project/raspberry-gpio-python/RPi.GPIO-${version}.tar.gz"
+checksum=7424bc6c205466764f30f666c18187a0824077daf20b295c42f08aea2cb87d3f
+
+# needed for gcc 10
+# fixed upstream in commit 8a0a65
+CFLAGS="-fcommon"
```

Comment:
Could ask upstream for a release, maybe? They have commits this year, last release was 2019...

Reply:
There is a pre-release [0.7.1a1](https://pypi.org/project/RPi.GPIO/0.7.1a4/#files) tarball that I think it's auto generated by pypy.org. Otherwise I will ask upstream for new release.

Thanks for merging this PR.